scripts/app.js

```diff
@@ -34,3 +39,19 @@ projectObjects.forEach(function(projectsObj) {
 projects.forEach(function(projectsObj) {
   $('#projects').append(projectsObj.toHtml());
 });
+
+Article.fetchAll = function() {
+  if (localStorage.projectObjStor) {
+    var projectObjStor = JSON.parse(localStorage.getItem('projectObjStor'));
+    Article.loadAll(projectObjStor);
+    console.log('data already in localStorage');
```

Comment:
If you're confident that the logic is behaving appropriately, you may also remove this console.log